### ISSUE TITLE
production bootstrap config for vaults

### ISSUE THREAD

**Issue by dckc** [Agoric/Agoric#7311]
## What is the Problem Being Solved?

similar to...
 - #6284

but for vaults instead of pismo

similar to
 - #6957 

but for production instead of testing

## Description of the Design

```[tasklist]
- [x] #7431
- [x] #7467
- [x] get production IBC denom for ATOM
```

nice to have:
 - #7309

## Security Considerations

This fixes the addresses of EC members (again) and initial Oracle operators.

## Scaling Considerations

can't thin of any

## Test Plan

_IOU_

**Comment by dckc** [comment]
Let's be sure to start `econCommitteeCharter` only once: https://github.com/Agoric/agoric-sdk/pull/7426#discussion_r1168855675

_For follow-up, see PR #7563 on issue #5819._

**Comment by dckc** [comment]
current devnet seems to have stakeFactory running; let's be sure that's not in the production config

_For follow-up, see..._
  - #7608

**Comment by otoole-brendan** [comment]
@dckc see mainnet addresses on #7431 which I have just closed

**Comment by ivanlei** [comment]
Per latest decision on release process, we intend to include Vaults/Oracles/Auction/Liquidation contracts in bootstrap but via governance leave the overall Vaults feature configured such that vault creation is not yet possible.

**Comment by dckc** [comment]
@arirubinstein and @JimLarson point out that the upgrade handler should cite this file

https://github.com/Agoric/agoric-sdk/blob/448d8276e893d75d24394b56e80f2f5fd8d7c54b/golang/cosmos/app/app.go#L784

_going back to draft_

**Comment by dckc** [comment]
https://github.com/Agoric/agoric-sdk/pull/7683 is some tweaks from discussion with @rowgraus 

@ivanlei feel free to assign this back to me if you want it to wai until tomorrow.

Cc @turadg

**Comment by ivanlei** [comment]
https://github.com/Agoric/agoric-sdk/pull/7683 has landed. Closing once again.
